app/services/zendesk_intake_service.rb

```diff
@@ -149,6 +149,26 @@ def send_final_intake_pdf
     output
   end
 
+  def send_intake_pdf_with_spouse
+    comment_body = <<~BODY
+      Updated 13614-c from online intake - added spouse signature and contact
```

Comment:
the completed intake comment includes a macro that sends a notification to clients. should we have one here?

Reply:
asked kelly